firebase-common/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 * [fixed] Correctly declare dependency on firebase-components, issue #5732
-
+* [changed] Added extension method `Random.nextAlphanumericString()` ([#5818](//github.com/firebase/firebase-android-sdk/pull/5818))
```

Comment:
Btw, you don't *have* to put the whole link. Our release tooling will automatically do it during release note generation. You can just do `#5818` (like the change entry above this one).

Of course, you're welcome to continue doing it regardless- the release note generation is smart enough to not overwrite it :)